app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: int,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self,
+                   car_list: list) -> float:
+        price = 0.0
+        for car in car_list:
+            if car.clean_mark < self.clean_power:
+                price += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return round(price, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = car.comfort_class * (self.clean_power - car.clean_mark) \
+            * (self.average_rating / self.distance_from_city_center)
```

Comment:
Add brackets on all your calculations and delete "\" from line 33.
Check checklist (Code style) for details.